poc/vdaf_prio3.py

```diff
@@ -483,6 +511,53 @@ class Prio3HistogramWithLength(Prio3Histogram):
         return Prio3HistogramWithLength
 
 
+# Experimental multiproof variant of Prio3SumVec
+class Prio3SumVecWithMultiproof(Prio3SumVec):
+    # Operational parameters.
+    test_vec_name = 'Prio3SumVecWithMultiproof'
+
+    @staticmethod
+    def is_recommended(valid_cls,
+                       num_proofs: Unsigned,
+                       field: FftField) -> bool:
+        if num_proofs >= 0x100:
+            # Currently we encod this as one byte while deriving joing rands
+            return False
+        # To be confirmed
```

Comment:
If this comment is referring to a decision that still needs to be made, then I'd make it more clear.
```suggestion
        # TODO(issue#177) Decide how many proofs to use for each field type.
```